### DISCUSSION TITLE
Erro no textarea

### DISCUSSION THREAD

**Discussion by callcocam** [wireui/wireui#115] [BUG]
@if ($name)
        <x-dynamic-component
            :component="WireUiComponent::resolve('icon')" <-- esta passando o icon mas parece que deve ser error
            :name="$name"
        />
    @endif

**Comment by ph7jack** [comment]
Hello @callcocam 

This error was resolved in the release [v0.13.4](https://github.com/PH7-Jack/wireui/releases/tag/v0.13.4)
